docs/install-config/configure-system-settings-cli.md

```diff
@@ -1,26 +1,27 @@
 ---
-title: Configure Harbor User Settings
+title: Harbor Configuration
 weight: 65
 ---
 
-User settings are configured separately from the [system settings](configure-yml-file.md). You can change user settings in the Harbor interface, through HTTP requests, or using an environment variable. This page describes the available user settings, and how to use the commandline or environment variable to update user settings.
+Harbor Configuration is configured separately from the [Configure the Harbor YML File](configure-yml-file.md) section. You can change the configuration in the Harbor interface, through HTTP requests, or using an environment variable. This page describes the available configuration items, and how to use the commandline or environment variable to update the configuration.
```

Comment:
Some Harbor configuration is configured separately from the [Configure the Harbor YML File](configure-yml-file.md) section...

Reply:
Updated